### PR TITLE
Productivity suite - small cleanup changes

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

**Review by petebacondarwin — APPROVED** [review]
Great clean up.

File: productivity-suite/app/fragments/news/src/root.tsx [hunk 1]
```diff
@@ -8,10 +8,10 @@ import {
   Scripts,
 } from "solid-start";
 import { PageEvent } from "solid-start/server";
-import { createContext, useContext } from "solid-js";
+import { createContext } from "solid-js";
 import "./root.css";
 
-export const ServerContext = createContext<PageEvent>({} as any);
+export const ServerContext = createContext<PageEvent>({} as PageEvent);
```
[the remark this answers is not in the document]
Comment (petebacondarwin) [reply productivity-suite/app/fragments/news/src/root.tsx:14]: But actually, I can't find any other use of this `ServerContext` export. So is it needed at all?

[the remark this answers is not in the document]
Comment (dario-piotrowicz) [reply productivity-suite/app/fragments/news/src/root.tsx:14]: let me check that.... honestly I just looked at the `any` there 🙈

[the remark this answers is not in the document]
Comment (dario-piotrowicz) [reply productivity-suite/app/fragments/news/src/root.tsx]: I think it is not needed, I've removed it and everything seems ok

I'm removing it, hopefully it's ok, if not we can always put it back 🙂

**Pull request merged**